go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/nspcc-dev/hrw/v2
 
-go 1.24
+go 1.24.0
```

Comment:
path num is inevitable now?

Reply:
Depends on dependencies, but since usually there are a lot of them it will come up. `X.Y.0` is OK to me, it's specified to have the same meaning as `X.Y`